tools/build/released_suites/PLATFORMS

```diff
@@ -16,6 +16,7 @@ tct-websocket-w3c-tests
 tct-xmlhttprequest-w3c-tests
 webapi-rawsockets-w3c-tests
 webapi-resourcetiming-w3c-tests
+webapi-serviceworkers-w3c-tests
```

Comment:
Because these tests shall be over HTTPS, I think we shall enable them after Web Testing Service supports HTTPS. See https://crosswalk-project.org/jira/browse/XWALK-6015

So please remove this line at current being.